src/com/yetanalytics/lrs_admin_ui/db.cljs

```diff
@@ -80,6 +80,57 @@
 (s/def ::oidc-auth boolean?)
 (s/def ::oidc-enable-local-admin boolean?)
 
+(s/def ::enable-admin-status boolean?)
+
+(s/def :status.data/statement-count nat-int?)
+(s/def :status.data/actor-count nat-int?)
+(s/def :status.data/last-statement-stored (s/nilable string?))
+;; The JSON->edn conversion makes the platform a keyword, so we handle that in
+;; the sub.
+(s/def :status.data/platform-frequency (s/map-of keyword? nat-int?))
+
+(s/def :status.data.timeline/stored string?)
+(s/def :status.data.timeline/count nat-int?)
+
+(s/def :status.data/timeline
+  (s/every
+   (s/keys :req-un [:status.data.timeline/stored
+                    :status.data.timeline/count])))
+
+(s/def :status/data
+  (s/keys :opt-un [:status.data/statement-count
+                   :status.data/actor-count
+                   :status.data/last-statement-stored
+                   :status.data/platform-frequency
+                   :status.data/timeline]))
+
+(s/def :status.params/timeline-unit
+  #{"year"
+    "month"
+    "day"
+    "hour"
+    "minute"
+    "second"})
+(s/def :status.params/timeline-since
+  string?)
+(s/def :status.params/timeline-until
+  string?)
+
+(s/def :status/params
+  (s/keys :opt-un [:status.params/timeline-unit
+                   :status.params/timeline-since
+                   :status.params/timeline-until]))
+
+;; map of vis type to loading state
+(s/def :status/loading
+  (s/map-of string? boolean?))
```

Comment:
I feel like this `string?` should be an enum spec, given that there is only a limited number of types of items to visualize.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/dc48ac489a9f314fbea66202569e0cea3e37884e

Reply:
Would've made the set a separate `def` but otherwise looks good.

Reply:
That enum doesn't need to be anywhere else and to my mind it is clearer to see what that map is w/o going somewhere else to read a def.